packages/components/button/src/Button.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropsWithChildren } from 'react'
+
+import { buttonStyles, type ButtonStylesProps } from './Button.styles'
+
+/** Review: Prop VS ButtonProps */
+interface Props
+  extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'size' | 'disabled'>,
+    ButtonStylesProps {}
+
+export const Button = React.forwardRef<HTMLButtonElement, PropsWithChildren<Props>>(
+  (
+    {
+      design = 'filled',
+      disabled = false,
+      intent = 'primary',
+      shape = 'rounded',
+      size = 'md',
+      ...rest
+    },
+    ref
+  ) => {
+    return (
+      <button
```

Comment:
I think that [slot](https://www.radix-ui.com/docs/primitives/utilities/slot) component should be used here. Normally you want anchors look like buttons. Here you have an [example](https://www.radix-ui.com/docs/primitives/utilities/slot#usage)

Reply:
yes. Its a primitive we might code soon